src/main/moditect/module-info.java

```diff
@@ -1,10 +1,9 @@
-module ezvcard {
+open module ezvcard {
```

Comment:
What happens if a user decides to exclude the JSON or HTML dependencies from their project because they do not need to use those vCard formats? (see: https://github.com/mangstadt/ez-vcard/wiki/Dependencies)

Reply:
Oh that's a good question, those should be probably marked as static then. I'll test it asap

Reply:
Fixed it with the latest commit.
I marked all of the dependencies you flagged in the pom as optional to be static.
(com.fasterxml.jackson.core is not static because it's not marked optional in the pom, only the databind is)